SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -836,28 +836,31 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 		if (SpatialGDKSettings->UseRPCRingBuffer() && RPCService != nullptr)
 		{
-			EPushRPCResult Result = RPCService->PushRPC(TargetObjectRef.Entity, RPCInfo.Type, Payload);
+			const EPushRPCResult Result = RPCService->PushRPC(TargetObjectRef.Entity, RPCInfo.Type, Payload);
+			const EPushRPCResult Outcome = PushRPCResultUtils::GetOutcome(Result);
+			const bool bQueued = PushRPCResultUtils::GetFailureAction(Result) == EPushRPCResult::Queue;
 #if !UE_BUILD_SHIPPING
-			if (Result == EPushRPCResult::Success || Result == EPushRPCResult::QueueOverflowed)
+			if (Outcome == EPushRPCResult::Success || bQueued)
 			{
 				TrackRPC(Channel->Actor, Function, Payload, RPCInfo.Type);
 			}
 #endif // !UE_BUILD_SHIPPING
 
-			switch (Result)
+			switch (Outcome)
```

Comment:
This comment is not for this PR, but why is logging code here? Could it be in `RPCContainer.cpp` instead, where similar RPC logging code live?

Reply:
This is logging specific to the ring buffers, so it makes sense for it to be next to the code it's concerning.